code/game/objects/structures/signs.dm

```diff
@@ -133,7 +133,7 @@
 /obj/structure/sign/greencross

```

Comment:
Change the object path aswell (and ensure to change it in all map files)

Reply:
done